docs/reference/migration/migrate_7_8.asciidoc

```diff
@@ -18,6 +19,26 @@ See also <<release-highlights>> and <<es-release-notes>>.
 
 //tag::notable-breaking-changes[]
 
+[discrete]
+[[breaking_781_license_changes]]
+=== License Information changes
+
+As of the `7.8.1` release
+
+.Displays enterprise license as platinum in /_xpack 
+[%collapsible]
+====
+*Details*
+The GET /_license endpoint displays "enterprise" licenses as
+"platinum" by default so that old clients (including beats, kibana and
+logstash) know to interpret this new license type as if it were a
+platinum license.
```

Comment:
```suggestion
The <<get-license,`GET /_license`>> endpoint displays Enterprise licenses as
Platinum by default so that old clients (including Beats, Kibana and
Logstash) know to interpret this new license type as if it were a
Platinum license.
```